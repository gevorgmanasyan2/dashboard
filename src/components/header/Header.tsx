import React, { FC } from 'react'
import Logo from './logo/Logo'
import Navbar from './navbar/Navbar'
import UserProfile from './user-profile/UserProfile'
import './header.scss'
import UserProfileMobile from './user-profile-mobile/UserProfileMobile'

const Header: FC = () => {
  return (
    <div className="header__container">
      <Logo />
      <Navbar />
      <div className="header__container__mobile">
        <UserProfile />
      </div>
      <UserProfileMobile />
    </div>
  )
}

export default Header
