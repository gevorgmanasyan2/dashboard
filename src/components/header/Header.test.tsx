/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'

it('should render Header component', () => {
  expect(shallow(<Header />)).toMatchSnapshot()
})
