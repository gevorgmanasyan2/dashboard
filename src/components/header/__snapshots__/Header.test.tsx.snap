// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<div
  className="header__container"
>
  <Logo />
  <Navbar />
  <div
    className="header__container__mobile"
  >
    <UserProfile />
  </div>
  <UserProfileMobile />
</div>
`;
